pom.xml

```diff
@@ -241,9 +241,8 @@
     </dependency>
     <dependency>
       <groupId>org.slf4j</groupId>
-      <artifactId>slf4j-simple</artifactId>
-      <version>1.7.36</version>
-      <scope>test</scope>
+      <artifactId>slf4j-api</artifactId>
+      <version>2.0.16</version>
```

Comment:
has no scope, is it possible to use test scope?

Reply:
Not possible to use test scope unless org.slf4j Logger gets provided via some route

DedupTransformation.java:[60,17] package org.slf4j does not exist
DedupTransformation.java:[68,26] cannot find symbol
[ERROR]   symbol:   class Logger
[ERROR]   location: class com.teragrep.pth10.ast.commands.transformstatement.DedupTransformation